src/sagas/communities/joinCommunity/joinCommunity.saga.ts

```diff
@@ -3,14 +3,12 @@ import { communitiesActions } from '../communities.slice';
 import { Socket } from 'socket.io-client';
 import { SocketActionTypes } from '../../socket/const/actionTypes';
 import { generateId } from '../../../utils/cryptography/cryptography';
+import { PayloadAction } from '@reduxjs/toolkit';
 
-export function* joinCommunitySaga(socket: Socket, action: any): Generator {
-  const id = yield call(generateId);
-  const payload = { id: id, registrarUrl: action.payload };
-  yield* put(communitiesActions.addNewCommunity(payload));
-  yield* put(communitiesActions.setCurrentCommunity(id));
-  yield* apply(socket, socket.emit, [
-    SocketActionTypes.CREATE_COMMUNITY,
-    { id },
-  ]);
-}
+export function* joinCommunitySaga (socket: Socket, action: PayloadAction<string>): Generator {
+    const id = yield* call(generateId)
+    const payload = {id: id, name: '', CA: {}, registrarUrl: action.payload}
+    yield* put(communitiesActions.addNewCommunity(payload))
```

Comment:
That's probably not a big deal but I would rather move updating local storage and marking current community after receiving confirmation of successfully storing it in database (if it doesn't much affect current logic ofc)

Reply:
We don't store anything in database here, because there is no database yet, even after creating database we don't store anything in it yet. I know that this is standard way when we work with waggle, but I am not sure about this particular case. I think we should return to this later.